workflow/models.py

```diff
@@ -1265,6 +1248,18 @@ def __unicode__(self):
         return unicode(self.workflowlevel1)
 
 
+class Product(models.Model):
+    workflowlevel2 = models.ForeignKey(WorkflowLevel2, related_name="products")
+    name = models.CharField(max_length=255)
+    type = models.CharField(max_length=255, blank=True, null=True)
+    number = models.CharField("Product identifier", max_length=255, blank=True, null=True)
```

Comment:
This name sounds too generic. It's better to use something like `external_id`.

Reply:
I was thinking the same. Maybe `reference_id`?

Reply:
Better than `number`